triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/TriggerRuleSimulationLib.scala

```diff
@@ -780,7 +780,7 @@ final class TriggerRuleSimulationLib private[simulation] (
           import GraphDSL.Implicits._
 
           val clientTime: Timestamp =
-            Timestamp.assertFromInstant(
+            Timestamp.assertLenientFromInstant(
```

Comment:
Can we make this strict, considering it's a test that we don't expose to end-users?